.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - "5.5"
-  - "4.2"
-  - "0.10"
```

Comment:
Not necessarily. It is still possible and not hard to install a more recent node.js version on Trusty by following https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions. The readme also mentions that you should run the latest LTS release.

I have no problem in leaving in older versions, but as these are EOL I'm reluctant to do anything if they start failing.